code/index.html

```diff
@@ -52,12 +55,13 @@ <h1>Does the person have</h1>
       <div class="win-or-lose">
         <img
           class="guess-who-icon"
-          src="images/guess-who-bubble.png"
+          src="images/SeekPng.com_guess-logo-png_1405949.png"
           alt="Guess Who"
         />
         <h1 id="winOrLoseText"></h1>
         <button id="playAgain" class="filled-button">PLAY AGAIN</button>
       </div>
```

Comment:
I can see you added the image to the win-or-lose section, but I didn't see it when I completed the game - so I'm guessing you didn't have time to finish the bonus step?